packages/manager/apps/hub/src/index.js

```diff
@@ -9,6 +10,8 @@ import ovhManagerCore from '@ovh-ux/manager-core';
 import ovhManagerHub from '@ovh-ux/manager-hub';
 import ovhManagerNavbar from '@ovh-ux/manager-navbar';
 
+import head from 'lodash/head';
```

Comment:
Looks like we need a plugin to reorder our imports

Reply:
Why reorder ? By alphabetical order ?

Reply:
Because everyone is doing it their own way, it's a mental overload we could automate, just like we did with linting and prettying

This looks like a start: https://eslint.org/docs/rules/sort-imports

This looks like it could help us: https://github.com/benmosher/eslint-plugin-import